WalletWasabi.Gui/Controls/WalletExplorer/WalletExplorerViewModel.cs

```diff
@@ -68,14 +71,31 @@ public WalletExplorerViewModel()
 					InsertWallet(vm);
 				});
 
+			CollapseAllCommand = ReactiveCommand.Create(CollapseWallets);
+
+			LurkingWifeModeCommand = ReactiveCommand.Create(ToggleLurkingWifeMode);
+
+			_isLurkingWifeMode = UiConfig
+				.WhenAnyValue(x => x.LurkingWifeMode)
+				.ToProperty(this, x => x.IsLurkingWifeMode, scheduler: RxApp.MainThreadScheduler);
+
+			LurkingWifeModeCommand.ThrownExceptions
```

Comment:
Use the usual pattern to Merge all commands Throw and handle them with one subscribe.

Reply:
done